lib/harvesting/models/invoice.rb

```diff
@@ -33,9 +33,18 @@ class Invoice < HarvestRecord
                  :created_at,
                  :updated_at
 
+      modeled client: Client,
+              creator: Client
```

Comment:
@chimedpalden I'm not familiar with this endpoint. Isn't the `creator` a `User`?